### PR TITLE
Update task

### PR DESCRIPTION
**Pull request by Trudko**

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -38,3 +39,19 @@ request.
...
+-------------
+
+To update a task call the [`tasks.update(taskID, options, callback)`](http://opensource.box.com/box-node-sdk/Tasks.html#update) method.
+
+```js
+client.tasks.update(
+	'1234',
+	{
+		message: 'Optional Message',
+		dueAt: '2018-04-03T11:09:43-07:00'
```
Comment (mattwiller) [inline docs/tasks.md]: With the new options passing style, this will need to be updated to `due_at`

File: lib/managers/tasks.js [hunk 2]
```diff
@@ -85,6 +85,39 @@ Tasks.prototype.get = function(taskID, qs, callback) {
...
+/**
+ * Updates a specific task.
+ *
+ * API Endpoint: '/tasks/:taskID'
+ * Method: PUT
+ *
+ * @param {string} taskID - The Box ID of the task being updated
+ * @param {?Object} options - Additional parameters
+ * @param {string} [options.message] - An optional message to include with the task
+ * @param {string} [options.dueAt] - The day at which this task is due
```
Comment (mattwiller) [inline lib/managers/tasks.js]: This will need to be updated to `due_at`

File: lib/managers/tasks.js [hunk 3]
```diff
@@ -85,6 +85,39 @@ Tasks.prototype.get = function(taskID, qs, callback) {
...
+			}
+		};
+
+	if (options && options.message) {
+		params.body.message = options.message;
+	}
+
+	if (options && options.dueAt) {
+		params.body.due_at = options.dueAt;
+	}
```
Comment (mattwiller) [inline lib/managers/tasks.js]: These will need to be updated to the standard options passing style, where all options are added to the body (not just whitelisted ones), e.g. by `Object.assign()`

File: docs/tasks.md (hunk 1 above)
Comment (Trudko) [inline docs/tasks.md]: Code updated.

File: lib/managers/tasks.js (hunk 2 above)
Comment (Trudko) [inline lib/managers/tasks.js]: Code updated.

**Pull request closed without merging**